mozfun/event_analysis/create_events_view/README.md

```diff
@@ -0,0 +1,181 @@
+### Usage
```

Comment:
Generally it's good for the levels of headings to create a hierarchy. Normally the first one would be a title, like:

```md
# Create Events View UDF
```

Is there a reason this wasn't done here?

Reply:
The generated doc structure does this for us. Basically we end up with one doc like the following:
```
# Event Analysis

...above text....

## create_events_view

...this is generated from the metdata.yaml...

### Usage

...this is this document here...
```

Now, I'm not opposed to dropping this README entirely and focusing all of `event_analysis/README.md` on this one stored procedure. However I'm assuming there will be others we want to publicly make available as well, all under the `event_analysis` namespace.

Reply:
Makes sense! As long as the end product has sensible header orderings I'm happy.